index.html

```diff
@@ -906,6 +902,27 @@ <h3>
         this attack.
       </p>
     </section>
+    <section id="same-origin-check">
```

Comment:
The Security section is declared non-normative and this is a subsection... maybe add it as a subsection of the "Process" section?

Reply:
good call